framework/src/components/README.md

```diff
@@ -0,0 +1,28 @@
+# Components
+
+## Description
+
+Components are shared objects within the [controller](../controller/README.md) layer which any [module](../modules/README.md) can utilize.
+Components can use [channels](../modules/README.md#module-communication) if required for implementation behavior.
+The following components are available currently.
+
+### Cache
+
+This component provides basic caching capabilities, generic enough for any module to use if required.
+
+### Logger
+
+Logger is responsible for all application-level logging activity and logs everything in JSON format.
+This central logger component can be passed to any module, where it can be extended by adding module-specific fields.
+
+### Storage
+
+This component is responsible for all database activity in the system.
+The database component exposes an interface with specific features for getting or setting particular database entities.
```

Comment:
```suggestion
It exposes an interface with specific features for getting or setting particular database entities and a raw handler to the database object so that any module can extend it for its own use.

Find more details about the storage component in the dedicated [LIP](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0011.md).
```